VMAccess/vmaccess.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
```

Comment:
If this approach is going to be used by other extensions, consider making this a more generic script that takes as argument the python script (with its arguments) that needs to be executed.

Reply:
Good idea. I added a new `-c` flag where one can give the python script. Please take a look at the update `$USAGE` for its usage.